grid-client/deployer/network_deployer.go

```diff
@@ -123,8 +218,18 @@ func (d *NetworkDeployer) GenerateVersionlessDeployments(ctx context.Context, zn
 	if err := znet.AssignNodesWGKey(allNodes); err != nil {
 		return nil, errors.Wrap(err, "could not assign node wg keys")
 	}
-	if err := znet.AssignNodesWGPort(ctx, sub, d.tfPluginClient.NcPool, allNodes, usedPorts); err != nil {
-		return nil, errors.Wrap(err, "could not assign node wg ports")
+	if znet.WGPort == nil {
+		znet.WGPort = make(map[uint32]int)
+	}
+	for _, nodeID := range allNodes {
+		nodeUsedPorts := usedPorts[nodeID]
```

Comment:
can we add a comment that we assign wg ports or move the following code to a new function?